config/overlays/mutation_webhook/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+# TODO: this is a temporary kustomization for the mutation webhook
+# It is kept separate until the mutation feature is stable enough,
+# when the mutation webhook should be moved to config/webhooks/manifests.yaml  
+
+namespace: gatekeeper-system
+
+bases:
+  - ../dev
+
+resources:
+  - webhook.yaml  
```

Comment:
nit: new line

Reply:
This belongs to another PR this depended on.
PR is not merged (with the newline)